hs_core/hydroshare/utils.py

```diff
@@ -672,30 +654,25 @@ def validate_user_quota(user_or_username, size):
         user = None
 
     if user:
-        # validate it is within quota hard limit
         uq = user.quotas.filter(zone='hydroshare').first()
+
         if uq:
             if not QuotaMessage.objects.exists():
                 QuotaMessage.objects.create()
             qmsg = QuotaMessage.objects.first()
-            enforce_flag = qmsg.enforce_quota
```

Comment:
enforce_quota property needs to be removed

Reply:
Wondering if this function should even be used at all or removed entirely, forcing us to be reliant completely on the quota system... I'm leaning towards the latter.